composer.json

```diff
@@ -15,12 +15,13 @@
         }
     },
     "require": {
-        "illuminate/support": "5.8.*|^6.0",
+        "php": "7.4 - 8.3",
```

Comment:
I'll be honest, `^7.4|^8.2` might be more beneficial, it would not allow `dev` versions and also be a more flexible

Reply:
The reasoning here was because `^8.2` would also allow `8.6` which I thought we would want to explicitly opt-in to since PHP itself doesn't follow semver.

Reply:
Fair point 😄